addons/web/views/webclient_templates.xml

```diff
@@ -439,7 +439,7 @@
         </xpath>
         <xpath expr="//t[@t-raw='0']" position="replace">
             <div id="wrapwrap" t-attf-class="#{pageName or ''}">
-                <header t-if="not no_header" id="top" data-anchor="true">
```

Comment:
We need that one. It is used for anchors. Adapt the JS to remove it from the duplicated one if that's the reason your removed it here.